src/components/Map/PlantingSiteMap.tsx

```diff
@@ -114,21 +113,18 @@ export default function PlantingSiteMap(props: PlantingSiteMapProps): JSX.Elemen
             properties: { id, name, fullName, type: 'plot' },
             boundary: getPolygons(boundary),
             id: `plot-${id}`,
-            fillColor,
-            fillOpacity,
-            lineColor,
-            lineWidth,
+            ...renderAttributes,
             isInteractive: true,
             annotation: {
               textField: 'fullName',
-              textColor: fillColor,
-              textSize: 10, // no tokens available yet
+              textColor: theme.palette.TwClrBaseWhite as string,
```

Comment:
plot label color

Reply:
Why are you using `as string` here and not `|| 'white'` as you did in other places?

Reply:
theme.palette.TwClrBaseWhite can be a string | undefined
mapbox requires a string
i removed the '|| 'white' because it wasn't as clean

Reply:
assumption here is that the theme color exists